### ISSUE TITLE
error when trying to test aria on form element

### ISSUE THREAD

**Issue by stephyswe** [NickColley/jest-axe#215]
Reproduce: CRA with jest-axe, add the following test file.

```jsx
import React from 'react'
import {render} from '@testing-library/react'
import {axe, toHaveNoViolations} from 'jest-axe'
import '@testing-library/jest-dom'

expect.extend(toHaveNoViolations)

function InnerForm() {
  return (
    <form>
      <input placeholder="email" />
    </form>
  )
}

/* test('view the form innerHTML', () => {
  const {container} = render(<Form />)
  //console.log(container.innerHTML)
}) */

describe('Some tests', () => {
  beforeAll(() => {
    // JSDom does not implement this and an error was being
    // thrown from jest-axe because of it.
    const {getComputedStyle} = window
    window.getComputedStyle = (elt) => getComputedStyle(elt)
  })

  test('the form is accessible', async () => {
    const {container} = render(<InnerForm />)
    const results = await axe(container.innerHTML)
    expect(results).toHaveNoViolations()
  })
})

```

Error: 

Some tests › the form is accessible

    expect(received).toHaveNoViolations(expected)

    Expected the HTML found at $('form') to have no violations:

    <form><input placeholder="email"></form>

    Received:

    "All page content should be contained by landmarks (region)"

    Fix any of the following:
      Some page content is not contained by landmarks

    You can find more information on this issue here:
    https://dequeuniversity.com/rules/axe/4.4/region?application=axeAPI

Tried a fix after suggestion from reactiflux

```
describe('Some tests', () => {
  beforeAll(() => {
    // JSDom does not implement this and an error was being
    // thrown from jest-axe because of it.
    const {getComputedStyle} = window
    window.getComputedStyle = (elt) => getComputedStyle(elt)
  })

  test('the form is accessible', async () => {
    const containerEl = document.createElement('div')
    containerEl.setAttribute('role', 'main')
    const view = render(<InnerForm />, {container: containerEl})
    const results = await axe(view.container)
    expect(results).toHaveNoViolations()
  })
})

```

But that just gave a green test, no aria checking.
Also removing window gives the window error again. 

Anyone know the fix? That probably hundreds of companies are using today? :)

**Comment by stephyswe** [comment]
Tried a fix after suggestion from reactiflux

describe('Some tests', () => {
  beforeAll(() => {
    // JSDom does not implement this and an error was being
    // thrown from jest-axe because of it.
    const {getComputedStyle} = window
    window.getComputedStyle = (elt) => getComputedStyle(elt)
  })

  test('the form is accessible', async () => {
    const containerEl = document.createElement('div')
    containerEl.setAttribute('role', 'main')
    const view = render(<InnerForm />, {container: containerEl})
    const results = await axe(view.container)
    expect(results).toHaveNoViolations()
  })
})


But that just gave a green test, no aria checking.
Also removing window gives the window error again. 

Anyone know the fix? That probably hundreds of companies are using today? :)

**Comment by mikeLspohn** [comment]
I'm having the same issue mentioned at the end where it just gives a green test with no checking. If I use a custom made `Input` component with no associated labels it passes, but if I use a regular `input` element with no label then the test fails as it should. Doing a `console.log(container.innerHTML)` before the expect everything looks correct in the HTML that gets logged.

**Comment by mikeLspohn** [comment]
Ok, so I found if I remove the `placeholder` attribute from my input being rendered by my Input component, THEN it starts failing with all the proper failures. So something with having placeholder present makes it seem like Labels are no longer present. Maybe that's just the case with a11y and I didn't realize that?

**Comment by NickColley** [comment]
Just published 7.0.0 please try your issue in 7.0.0 and re-open if it's an issue, thanks a lot.
